tests/functional/base_test.go

```diff
@@ -274,6 +278,55 @@ func CreateOpenStackControlPlane(name types.NamespacedName, spec map[string]inte
 	return th.CreateUnstructured(raw)
 }
 
+// Build OpenStackDataPlaneNodeSetSpec struct with empty `Nodes` list
+func DefaultDataPlaneNoNodeSetSpec(tlsEnabled bool) map[string]interface{} {
```

Comment:
@fao89 hey, just a heads up on the dataplane-operator merge I needed to forklift a few functions here to implement some functional tests for the minor updates workflow.